unnecessary translation warning displayed in script

Copied from DM

`04-sb\eureka\eureka_hydatos.js:88 BA Seal missing timelineReplace for regex '^^(?:00)\|(?:[^|]*)\|(?:0839)\|(?:[^|]*)\|(?:.* will be sealed off.*?)\|' (but have netRegexDe)` is the last entry for german translations missing, but whenever i add this `'.* will be sealed off.*?': 'Noch 15 Sekunden, bis sich (?:(?:der|die|das) )?(?:Zugang zu(?:[rm]| den)? )?.* schließt.*?',` and run the test, i get this

```
 1) timeline test
       ui/raidboss/data/04-sb/eureka/eureka_hydatos.txt
         should not have translation conflicts:

      AssertionError: ui/raidboss/data/04-sb/eureka/eureka_hydatos.js:locale de: pre-translation collision on replaceSync '00:0839:The Shin-Zantetsuken Containment Unit will be sealed off' for '/.* will be sealed off.*?/i' and '/(?<=00:0839:)(.*) will be sealed off(?: in (?:[0-9]+ seconds)?)?/i': expected '00:0839:Noch 15 Sekunden, bis sich (?:(?:der|die|das) )?(?:Zugang zu(?:[rm]| den)? )?The Shin-Zantetsuken Containment Unit schließt' to equal 'Noch 15 Sekunden, bis sich (?:(?:der|die|das) )?(?:Zugang zu(?:[rm]| den)? )?.* schließt.*?'
      + expected - actual

      -00:0839:Noch 15 Sekunden, bis sich (?:(?:der|die|das) )?(?:Zugang zu(?:[rm]| den)? )?The Shin-Zantetsuken Containment Unit schließt
      +Noch 15 Sekunden, bis sich (?:(?:der|die|das) )?(?:Zugang zu(?:[rm]| den)? )?.* schließt.*?
```

I just rebased and when i add the translation to hydatos i get following:

```
  1 failing

  1) timeline test
       ui/raidboss/data/04-sb/eureka/eureka_hydatos.txt
         should not have translation conflicts:

      AssertionError: ui/raidboss/data/04-sb/eureka/eureka_hydatos.js:locale de: pre-translation collision on replaceSync '00:0839:The Shin-Zantetsuken Containment Unit will be sealed off' for '/.* will be sealed off.*?/i' and '/(?<=00:0839:)(.*) will be sealed off(?: in (?:[0-9]+ seconds)?)?/i': expected '00:0839:Noch 15 Sekunden, bis sich (?:(?:der|die|das) )?(?:Zugang zu(?:[rm]| den)? )?The Shin-Zantetsuken Containment Unit schließt' to equal 'Noch 15 Sekunden, bis sich (?:(?:der|die|das) )?(?:Zugang zu(?:[rm]| den)? )?.* schließt.*?'
      + expected - actual

      -00:0839:Noch 15 Sekunden, bis sich (?:(?:der|die|das) )?(?:Zugang zu(?:[rm]| den)? )?The Shin-Zantetsuken Containment Unit schließt
      +Noch 15 Sekunden, bis sich (?:(?:der|die|das) )?(?:Zugang zu(?:[rm]| den)? )?.* schließt.*?
```

--- a/ui/raidboss/common_replacement.ts
+++ b/ui/raidboss/common_replacement.ts
@@ -2,7 +2,8 @@
 
 // It's awkward to refer to these string keys, so name them as replaceSync[keys.sealKey].
 export const syncKeys = {
-  seal: '(?<=00:0839:)(.*) will be sealed off(?: in (?:[0-9]+ seconds)?)?',
+  // Match Regexes, NetRegexes, and timeline constructions of seal log lines.
+  seal: '(?<=00:0839:|00.*0839.*\\\\\|)(.*) will be sealed off(?: in (?:[0-9]+ seconds)?)?',
   unseal: 'is no longer sealed',
   engage: 'Engage!',
 };

